content/docs/rfcs/31/README.md

```diff
@@ -118,8 +118,42 @@ Supported key type is `secp256k1` only.
 
 In the future, an extension of this RFC could be made to support other elliptic curve cryptography such as `ed25519`.
 
+# `waku2` ENR key
+
+We propose the definition of a `waku2` field key:
+
+- The value MUST be an 8-bit flag field,
+where bits set to `1` indicate `true` and bits set to `0` indicate `false` for the relevant flags.
+- The flag values already defined are set out below,
+with `bit 7` the most significant bit and `bit 0` the least significant bit.
```

Comment:
Any considerations wrt endianness here and how it would look different in different clients?

Reply:
Wouldn't that simply affect how these are packed/unpacked in memory for different systems? In other words, bit "definitions" from MSB -> LSB remains the same, even if in big-endian systems bits 7-4 are packed first and in little-endian bits 3-0 are packed first.